SmartDeviceLink/SDLRPCResponseNotification.m

```diff
@@ -30,4 +30,9 @@ - (__kindof SDLRPCResponse *)response {
     return _userInfo[SDLNotificationUserInfoObject];
 }
 
+- (BOOL)isResponseKindOfClass:(Class)aClass {
+    NSAssert([self.response isKindOfClass:aClass], @"A notification was sent with an unanticipated object");
```

Comment:
I added in `isMemberOfClass`, however internally we have a call in `SDLResponseDispatcher` that actually requires `isKindOfClass`, so I kept both. I'm not super keen on that, so open to thoughts.